{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "cfac68ca",
   "metadata": {},
   "source": [
    "# Download fasta sequences\n",
    "Go to www.ensembl.org to find the gene id for the gene name TMPRSS2 and ERG"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "bd5cb13a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100 52571  100 52571    0     0   107k      0 --:--:-- --:--:-- --:--:--  107k\n",
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100 50291  100 50291    0     0  97652      0 --:--:-- --:--:-- --:--:-- 97463\n",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "#TMPRSS2 \n",
    "curl 'https://rest.ensembl.org/sequence/id/ENSG00000184012?type=cdna;multiple_sequences=100' -H 'Content-type:text/x-fasta' > TMPRSS2.fasta\n",
    "\n",
    "#ERG\n",
    "curl 'https://rest.ensembl.org/sequence/id/ENSG00000157554?type=cdna;multiple_sequences=100' -H 'Content-type:text/x-fasta' > ERG.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a8c2a064",
   "metadata": {},
   "source": [
    "## Manually create the fusions\n",
    "[Cosmic Gene Fusion](https://cancer.sanger.ac.uk/cosmic/fusion/overview?fid=250888&gid=291726) to find breakpoints <br>\n",
    "TMPRSS2_ENST00000332149, ERG_ENST00000442448 transcripts used for fusion <br>\n",
    "- exon 2 from TMPRSS2 and end of exon 4 from ERG\n",
    "- exon 3 from TMPRSS2 and beginning of exon 4 from ERG\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4b8a3e96",
   "metadata": {},
   "source": [
    "# Create input fasta with defined wild type and fusion expression"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "1ef3d4d9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?2004l\r",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "mkdir WildType_Fusion_Quantify"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "38a3b655",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "# low wild type expression high fusion expression\n",
    "for i in {1..1};do cat TMPRSS2_ENST00000332149.fasta; done > T_TMP.fasta\n",
    "for i in {1..1};do cat ERG_ENST00000442448.fasta; done > E_TMP.fasta\n",
    "for i in {1..4};do cat TMPRSS2_ERG_ManuallyFused.fasta; done > F_TMP.fasta\n",
    "cat T_TMP.fasta E_TMP.fasta F_TMP.fasta >  WildType_Fusion_Quantify/LowWT_HighFusion.fasta\n",
    "rm T_TMP.fasta E_TMP.fasta F_TMP.fasta\n",
    "\n",
    "# low wild type expression high fusion expression\n",
    "for i in {1..4};do cat TMPRSS2_ENST00000332149.fasta; done > T_TMP.fasta\n",
    "for i in {1..4};do cat ERG_ENST00000442448.fasta; done > E_TMP.fasta\n",
    "for i in {1..1};do cat TMPRSS2_ERG_ManuallyFused.fasta; done > F_TMP.fasta\n",
    "cat T_TMP.fasta E_TMP.fasta F_TMP.fasta >  WildType_Fusion_Quantify/HighWT_LowFusion.fasta\n",
    "rm T_TMP.fasta E_TMP.fasta F_TMP.fasta\n",
    "\n",
    "# similar wild type expression and  fusion expression\n",
    "for i in {1..3};do cat TMPRSS2_ENST00000332149.fasta; done > T_TMP.fasta\n",
    "for i in {1..3};do cat ERG_ENST00000442448.fasta; done > E_TMP.fasta\n",
    "for i in {1..2};do cat TMPRSS2_ERG_ManuallyFused.fasta; done > F_TMP.fasta\n",
    "cat T_TMP.fasta E_TMP.fasta F_TMP.fasta >  WildType_Fusion_Quantify/SimilarWT_Fusion.fasta\n",
    "rm T_TMP.fasta E_TMP.fasta F_TMP.fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "6ac12d9e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000332149.10\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000442448.5\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon2_ERG-exon4\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon3_ERG-exon4\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon2_ERG-exon4\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon3_ERG-exon4\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon2_ERG-exon4\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon3_ERG-exon4\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon2_ERG-exon4\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon3_ERG-exon4\n",
      "\u001b[?2004h\u001b[?2004l\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000332149.10\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000332149.10\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000332149.10\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000332149.10\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000442448.5\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000442448.5\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000442448.5\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000442448.5\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon2_ERG-exon4\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon3_ERG-exon4\n",
      "\u001b[?2004h\u001b[?2004l\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000332149.10\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000332149.10\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000332149.10\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000442448.5\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000442448.5\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KENST00000442448.5\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon2_ERG-exon4\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon3_ERG-exon4\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon2_ERG-exon4\n",
      "\u001b[01;31m\u001b[K>\u001b[m\u001b[KTMPRSS2-exon3_ERG-exon4\n",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "grep \">\" WildType_Fusion_Quantify/LowWT_HighFusion.fasta\n",
    "echo \"\"\n",
    "grep \">\" WildType_Fusion_Quantify/HighWT_LowFusion.fasta\n",
    "echo \"\"\n",
    "grep \">\" WildType_Fusion_Quantify/SimilarWT_Fusion.fasta"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fdc13dcf",
   "metadata": {},
   "source": [
    "# Simulate reads from FASTA seq\n",
    "There are 10 transcripts in the input fasta representing 3 genes(2 wild type and 1 fusion).\n",
    "A rough estimate is 10000 reads per transcript, if an rnaseq experiment is 200M reads for 20,000 transcripts.\n",
    "Hence art simulates 3000 reads from each transcript resulting in 30,000 reads in total"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "be5c1d34",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.7 (Mar 19, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1.01\n",
      "\n",
      "The random seed for the run: 3\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0X\n",
      "\tMean Fragment Length:     200\n",
      "\tStandard Deviation:       10\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   HiSeq 2500 Length 150 R1 (built-in profile) \n",
      "\tFirst Read:   HiSeq 2500 Length 150 R2 (built-in profile) \n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: HighWT_LowFusion1.fq\n",
      "\t the 2nd reads: HighWT_LowFusion2.fq\n",
      "\n",
      "  ALN Alignment Files:\n",
      "\t the 1st reads: HighWT_LowFusion1.aln\n",
      "\t the 2nd reads: HighWT_LowFusion2.aln\n",
      "\n",
      "  SAM Alignment File:\n",
      "\tHighWT_LowFusion.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.7 (Mar 19, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1\n",
      "\n",
      "The random seed for the run: 3\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0X\n",
      "\tMean Fragment Length:     200\n",
      "\tStandard Deviation:       10\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   HiSeq 2500 Length 150 R1 (built-in profile) \n",
      "\tFirst Read:   HiSeq 2500 Length 150 R2 (built-in profile) \n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: LowWT_HighFusion1.fq\n",
      "\t the 2nd reads: LowWT_HighFusion2.fq\n",
      "\n",
      "  ALN Alignment Files:\n",
      "\t the 1st reads: LowWT_HighFusion1.aln\n",
      "\t the 2nd reads: LowWT_HighFusion2.aln\n",
      "\n",
      "  SAM Alignment File:\n",
      "\tLowWT_HighFusion.sam\n",
      "\n",
      "=======================================ART======================================\n",
      "                            ART_Illumina (2008-2015)                            \n",
      "                            Q Version 2.3.7 (Mar 19, 2015)                      \n",
      "                     Contact: Weichun Huang at whduke@gmail.com                 \n",
      "================================================================================\n",
      "\n",
      "                          Paired-end sequencing simulation\n",
      "\n",
      "Total CPU time used: 1\n",
      "\n",
      "The random seed for the run: 3\n",
      "\n",
      "Parameters used during run\n",
      "\tRead Length:\t150\n",
      "\tGenome masking 'N' cutoff frequency: \t1 in 150\n",
      "\tFold Coverage:            0X\n",
      "\tMean Fragment Length:     200\n",
      "\tStandard Deviation:       10\n",
      "\tProfile Type:             Combined\n",
      "\tID Tag:                   \n",
      "\n",
      "Quality Profile(s)\n",
      "\tFirst Read:   HiSeq 2500 Length 150 R1 (built-in profile) \n",
      "\tFirst Read:   HiSeq 2500 Length 150 R2 (built-in profile) \n",
      "Output files\n",
      "\n",
      "  FASTQ Sequence Files:\n",
      "\t the 1st reads: SimilarWT_Fusion1.fq\n",
      "\t the 2nd reads: SimilarWT_Fusion2.fq\n",
      "\n",
      "  ALN Alignment Files:\n",
      "\t the 1st reads: SimilarWT_Fusion1.aln\n",
      "\t the 2nd reads: SimilarWT_Fusion2.aln\n",
      "\n",
      "  SAM Alignment File:\n",
      "\tSimilarWT_Fusion.sam\n",
      "\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "art=/home/jp/anaconda3/envs/jpEnv/bin/art_illumina\n",
    "\n",
    "$art -sam -i WildType_Fusion_Quantify/HighWT_LowFusion.fasta -p -l 150 -ss HS25 -c 3000 -m 200 -s 10 -rs 3 -o HighWT_LowFusion\n",
    "$art -sam -i WildType_Fusion_Quantify/LowWT_HighFusion.fasta -p -l 150 -ss HS25 -c 3000 -m 200 -s 10 -rs 3 -o LowWT_HighFusion\n",
    "$art -sam -i WildType_Fusion_Quantify/SimilarWT_Fusion.fasta -p -l 150 -ss HS25 -c 3000 -m 200 -s 10 -rs 3 -o SimilarWT_Fusion\n",
    "\n",
    "\n",
    "rm *.sam *.aln\n",
    "\n",
    "mv *.fq WildType_Fusion_Quantify"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "f5878358",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "120000 WildType_Fusion_Quantify/HighWT_LowFusion1.fq\n",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "wc -l WildType_Fusion_Quantify/HighWT_LowFusion1.fq"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fa822d54",
   "metadata": {},
   "source": [
    "# Downsampling FASTQ files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "0d277548",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?2004l\r",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "seqtk=/home/jp/anaconda3/envs/jpEnv/bin/seqtk"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "8913439b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l\u001b[?2004l"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "# downsample 50%\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/HighWT_LowFusion1.fq 0.5 > WildType_Fusion_Quantify/HighWT_LowFusion1_0.5.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/HighWT_LowFusion2.fq 0.5 > WildType_Fusion_Quantify/HighWT_LowFusion2_0.5.fq\n",
    "\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/LowWT_HighFusion1.fq 0.5 > WildType_Fusion_Quantify/LowWT_HighFusion1_0.5.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/LowWT_HighFusion2.fq 0.5 > WildType_Fusion_Quantify/LowWT_HighFusion2_0.5.fq\n",
    "\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/SimilarWT_Fusion1.fq 0.5 > WildType_Fusion_Quantify/SimilarWT_Fusion1_0.5.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/SimilarWT_Fusion2.fq 0.5 > WildType_Fusion_Quantify/SimilarWT_Fusion2_0.5.fq\n",
    "\n",
    "\n",
    "\n",
    "# downsample 40%\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/HighWT_LowFusion1.fq 0.4 > WildType_Fusion_Quantify/HighWT_LowFusion1_0.4.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/HighWT_LowFusion2.fq 0.4 > WildType_Fusion_Quantify/HighWT_LowFusion2_0.4.fq\n",
    "\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/LowWT_HighFusion1.fq 0.4 > WildType_Fusion_Quantify/LowWT_HighFusion1_0.4.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/LowWT_HighFusion2.fq 0.4 > WildType_Fusion_Quantify/LowWT_HighFusion2_0.4.fq\n",
    "\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/SimilarWT_Fusion1.fq 0.4 > WildType_Fusion_Quantify/SimilarWT_Fusion1_0.4.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/SimilarWT_Fusion2.fq 0.4 > WildType_Fusion_Quantify/SimilarWT_Fusion2_0.4.fq\n",
    "\n",
    "\n",
    "\n",
    "# downsample 30%\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/HighWT_LowFusion1.fq 0.3 > WildType_Fusion_Quantify/HighWT_LowFusion1_0.3.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/HighWT_LowFusion2.fq 0.3 > WildType_Fusion_Quantify/HighWT_LowFusion2_0.3.fq\n",
    "\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/LowWT_HighFusion1.fq 0.3 > WildType_Fusion_Quantify/LowWT_HighFusion1_0.3.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/LowWT_HighFusion2.fq 0.3 > WildType_Fusion_Quantify/LowWT_HighFusion2_0.3.fq\n",
    "\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/SimilarWT_Fusion1.fq 0.3 > WildType_Fusion_Quantify/SimilarWT_Fusion1_0.3.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/SimilarWT_Fusion2.fq 0.3 > WildType_Fusion_Quantify/SimilarWT_Fusion2_0.3.fq\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "# downsample 20%\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/HighWT_LowFusion1.fq 0.2 > WildType_Fusion_Quantify/HighWT_LowFusion1_0.2.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/HighWT_LowFusion2.fq 0.2 > WildType_Fusion_Quantify/HighWT_LowFusion2_0.2.fq\n",
    "\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/LowWT_HighFusion1.fq 0.2 > WildType_Fusion_Quantify/LowWT_HighFusion1_0.2.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/LowWT_HighFusion2.fq 0.2 > WildType_Fusion_Quantify/LowWT_HighFusion2_0.2.fq\n",
    "\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/SimilarWT_Fusion1.fq 0.2 > WildType_Fusion_Quantify/SimilarWT_Fusion1_0.2.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/SimilarWT_Fusion2.fq 0.2 > WildType_Fusion_Quantify/SimilarWT_Fusion2_0.2.fq\n",
    "\n",
    "\n",
    "\n",
    "# downsample 10%\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/HighWT_LowFusion1.fq 0.1 > WildType_Fusion_Quantify/HighWT_LowFusion1_0.1.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/HighWT_LowFusion2.fq 0.1 > WildType_Fusion_Quantify/HighWT_LowFusion2_0.1.fq\n",
    "\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/LowWT_HighFusion1.fq 0.1 > WildType_Fusion_Quantify/LowWT_HighFusion1_0.1.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/LowWT_HighFusion2.fq 0.1 > WildType_Fusion_Quantify/LowWT_HighFusion2_0.1.fq\n",
    "\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/SimilarWT_Fusion1.fq 0.1 > WildType_Fusion_Quantify/SimilarWT_Fusion1_0.1.fq\n",
    "$seqtk sample -s3 WildType_Fusion_Quantify/SimilarWT_Fusion2.fq 0.1 > WildType_Fusion_Quantify/SimilarWT_Fusion2_0.1.fq\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "b41967fd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "12160 WildType_Fusion_Quantify/LowWT_HighFusion1_0.1.fq\n",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "wc -l WildType_Fusion_Quantify/LowWT_HighFusion1_0.1.fq"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "82502134",
   "metadata": {},
   "source": [
    "# Run STAR-Fusion"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "5be8a18e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?2004l\r",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "mkdir StarFusion"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "7db8dcbd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "gen=/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
    "starFusion=/home/jp/STAR-Fusion-v1.10.0/STAR-Fusion\n",
    "thread=10"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e5edefe4",
   "metadata": {},
   "source": [
    "## High WT Low Fusion"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "aff04be6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "-- Skipping CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion1.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion2.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts, checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/run_star_aligner.ok] exists.\n",
      "-sample contains 30000 rnaseq fragments\n",
      "-- Skipping CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/star-fusion.map_chim_to_genes.ok] exists.\n",
      "-- Skipping CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass , checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/star-fusion.handle_multimappers.ok] exists.\n",
      "-- Skipping CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion , checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/star-fusion.STAR-Fusion.predict.ok] exists.\n",
      "-- Skipping CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion, checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/STAR-Fusion.filter.ok] exists.\n",
      "-- Skipping CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 30000 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM, checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/FFPM.ok] exists.\n",
      "-- Skipping CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo, checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/predicted_fusions_with_splice_info.ok] exists.\n",
      "-- Skipping CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot, checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/fusion_annotator.ok] exists.\n",
      "-- Skipping CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir, checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/annot_filter.ok] exists.\n",
      "-- Skipping CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir, checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/RTartifact_filter.ok] exists.\n",
      "-- Skipping CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1, checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/FFPM_filter.ok] exists.\n",
      "-- Skipping CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv, checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/cp_final.ok] exists.\n",
      "-- Skipping CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv, checkpoint [/home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/_starF_checkpoints/abridged_final] exists.\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion1_0.5.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion2_0.5.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion1_0.5.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion2_0.5.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 09:51:14 ..... started STAR run\n",
      "Jun 08 09:51:14 ..... loading genome\n",
      "Jun 08 09:51:22 ..... started 1st pass mapping\n",
      "Jun 08 09:51:26 ..... finished 1st pass mapping\n",
      "Jun 08 09:51:26 ..... inserting junctions into the genome indices\n",
      "Jun 08 09:52:24 ..... started mapping\n",
      "Jun 08 09:52:29 ..... finished mapping\n",
      "Jun 08 09:52:30 ..... finished successfully\n",
      "-sample contains 15208 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n",
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -42.208178\n",
      "EM: Round [1] log likelihood: -42.208178\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n",
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 15208 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion1_0.4.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion2_0.4.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion1_0.4.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion2_0.4.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 09:52:35 ..... started STAR run\n",
      "Jun 08 09:52:35 ..... loading genome\n",
      "Jun 08 09:52:42 ..... started 1st pass mapping\n",
      "Jun 08 09:52:46 ..... finished 1st pass mapping\n",
      "Jun 08 09:52:46 ..... inserting junctions into the genome indices\n",
      "Jun 08 09:53:45 ..... started mapping\n",
      "Jun 08 09:53:48 ..... finished mapping\n",
      "Jun 08 09:53:49 ..... finished successfully\n",
      "-sample contains 12120 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n",
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -30.453006\n",
      "EM: Round [1] log likelihood: -30.453006\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n",
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 12120 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion1_0.3.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion2_0.3.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion1_0.3.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion2_0.3.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 09:53:54 ..... started STAR run\n",
      "Jun 08 09:53:54 ..... loading genome\n",
      "Jun 08 09:54:02 ..... started 1st pass mapping\n",
      "Jun 08 09:54:04 ..... finished 1st pass mapping\n",
      "Jun 08 09:54:05 ..... inserting junctions into the genome indices\n",
      "Jun 08 09:55:04 ..... started mapping\n",
      "Jun 08 09:55:07 ..... finished mapping\n",
      "Jun 08 09:55:08 ..... finished successfully\n",
      "-sample contains 9137 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n",
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -26.128676\n",
      "EM: Round [1] log likelihood: -26.128676\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 9137 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion1_0.2.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion2_0.2.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion1_0.2.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion2_0.2.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 09:55:13 ..... started STAR run\n",
      "Jun 08 09:55:13 ..... loading genome\n",
      "Jun 08 09:55:20 ..... started 1st pass mapping\n",
      "Jun 08 09:55:22 ..... finished 1st pass mapping\n",
      "Jun 08 09:55:23 ..... inserting junctions into the genome indices\n",
      "Jun 08 09:56:21 ..... started mapping\n",
      "Jun 08 09:56:24 ..... finished mapping\n",
      "Jun 08 09:56:24 ..... finished successfully\n",
      "-sample contains 6113 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -18.021827\n",
      "EM: Round [1] log likelihood: -18.021827\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n",
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 6113 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/_starF_checkpoints\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion1_0.1.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion2_0.1.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion1_0.1.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/HighWT_LowFusion2_0.1.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 09:56:29 ..... started STAR run\n",
      "Jun 08 09:56:29 ..... loading genome\n",
      "Jun 08 09:56:37 ..... started 1st pass mapping\n",
      "Jun 08 09:56:38 ..... finished 1st pass mapping\n",
      "Jun 08 09:56:38 ..... inserting junctions into the genome indices\n",
      "Jun 08 09:57:37 ..... started mapping\n",
      "Jun 08 09:57:38 ..... finished mapping\n",
      "Jun 08 09:57:39 ..... finished successfully\n",
      "-sample contains 3040 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n",
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -8.150319\n",
      "EM: Round [1] log likelihood: -8.150319\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 3040 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "p1=WildType_Fusion_Quantify/HighWT_LowFusion1.fq\n",
    "p2=WildType_Fusion_Quantify/HighWT_LowFusion2.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/HighWT_LowFusion --CPU $thread\n",
    "\n",
    "\n",
    "# 50% downsampling\n",
    "p1=WildType_Fusion_Quantify/HighWT_LowFusion1_0.5.fq\n",
    "p2=WildType_Fusion_Quantify/HighWT_LowFusion2_0.5.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/HighWT_LowFusion_0.5 --CPU $thread\n",
    "\n",
    "\n",
    "# 40% downsampling\n",
    "p1=WildType_Fusion_Quantify/HighWT_LowFusion1_0.4.fq\n",
    "p2=WildType_Fusion_Quantify/HighWT_LowFusion2_0.4.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/HighWT_LowFusion_0.4 --CPU $thread\n",
    "\n",
    "\n",
    "# 30% downsampling\n",
    "p1=WildType_Fusion_Quantify/HighWT_LowFusion1_0.3.fq\n",
    "p2=WildType_Fusion_Quantify/HighWT_LowFusion2_0.3.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/HighWT_LowFusion_0.3 --CPU $thread\n",
    "\n",
    "\n",
    "# 20% downsampling\n",
    "p1=WildType_Fusion_Quantify/HighWT_LowFusion1_0.2.fq\n",
    "p2=WildType_Fusion_Quantify/HighWT_LowFusion2_0.2.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/HighWT_LowFusion_0.2 --CPU $thread\n",
    "\n",
    "\n",
    "# 10% downsampling\n",
    "p1=WildType_Fusion_Quantify/HighWT_LowFusion1_0.1.fq\n",
    "p2=WildType_Fusion_Quantify/HighWT_LowFusion2_0.1.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/HighWT_LowFusion_0.1 --CPU $thread\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "149711b7",
   "metadata": {},
   "source": [
    "## Low WT High Fusion"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "id": "4f792588",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 13:48:38 ..... started STAR run\n",
      "Jun 08 13:48:38 ..... loading genome\n",
      "Jun 08 13:48:46 ..... started 1st pass mapping\n",
      "Jun 08 13:48:54 ..... finished 1st pass mapping\n",
      "Jun 08 13:48:54 ..... inserting junctions into the genome indices\n",
      "Jun 08 13:49:52 ..... started mapping\n",
      "Jun 08 13:50:02 ..... finished mapping\n",
      "Jun 08 13:50:03 ..... finished successfully\n",
      "-sample contains 30000 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n",
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -337.205891\n",
      "EM: Round [1] log likelihood: -337.205891\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 30000 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1_0.5.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2_0.5.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1_0.5.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2_0.5.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 13:50:08 ..... started STAR run\n",
      "Jun 08 13:50:08 ..... loading genome\n",
      "Jun 08 13:50:15 ..... started 1st pass mapping\n",
      "Jun 08 13:50:19 ..... finished 1st pass mapping\n",
      "Jun 08 13:50:20 ..... inserting junctions into the genome indices\n",
      "Jun 08 13:51:18 ..... started mapping\n",
      "Jun 08 13:51:23 ..... finished mapping\n",
      "Jun 08 13:51:24 ..... finished successfully\n",
      "-sample contains 15208 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -166.931663\n",
      "EM: Round [1] log likelihood: -166.931663\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n",
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 15208 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/_starF_checkpoints\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1_0.4.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2_0.4.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1_0.4.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2_0.4.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 13:51:29 ..... started STAR run\n",
      "Jun 08 13:51:29 ..... loading genome\n",
      "Jun 08 13:51:36 ..... started 1st pass mapping\n",
      "Jun 08 13:51:40 ..... finished 1st pass mapping\n",
      "Jun 08 13:51:40 ..... inserting junctions into the genome indices\n",
      "Jun 08 13:52:38 ..... started mapping\n",
      "Jun 08 13:52:43 ..... finished mapping\n",
      "Jun 08 13:52:43 ..... finished successfully\n",
      "-sample contains 12120 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.07 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n",
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -129.487476\n",
      "EM: Round [1] log likelihood: -129.487476\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 12120 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1_0.3.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2_0.3.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1_0.3.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2_0.3.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 13:52:48 ..... started STAR run\n",
      "Jun 08 13:52:48 ..... loading genome\n",
      "Jun 08 13:52:56 ..... started 1st pass mapping\n",
      "Jun 08 13:52:58 ..... finished 1st pass mapping\n",
      "Jun 08 13:52:59 ..... inserting junctions into the genome indices\n",
      "Jun 08 13:53:58 ..... started mapping\n",
      "Jun 08 13:54:01 ..... finished mapping\n",
      "Jun 08 13:54:02 ..... finished successfully\n",
      "-sample contains 9137 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -90.523342\n",
      "EM: Round [1] log likelihood: -90.523342\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n",
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 9137 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/_starF_checkpoints\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1_0.2.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2_0.2.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1_0.2.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2_0.2.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 13:54:07 ..... started STAR run\n",
      "Jun 08 13:54:07 ..... loading genome\n",
      "Jun 08 13:54:14 ..... started 1st pass mapping\n",
      "Jun 08 13:54:16 ..... finished 1st pass mapping\n",
      "Jun 08 13:54:17 ..... inserting junctions into the genome indices\n",
      "Jun 08 13:55:15 ..... started mapping\n",
      "Jun 08 13:55:17 ..... finished mapping\n",
      "Jun 08 13:55:18 ..... finished successfully\n",
      "-sample contains 6113 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.07 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n",
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -60.420001\n",
      "EM: Round [1] log likelihood: -60.420001\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 6113 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1_0.1.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2_0.1.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion1_0.1.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/LowWT_HighFusion2_0.1.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 13:55:23 ..... started STAR run\n",
      "Jun 08 13:55:23 ..... loading genome\n",
      "Jun 08 13:55:30 ..... started 1st pass mapping\n",
      "Jun 08 13:55:32 ..... finished 1st pass mapping\n",
      "Jun 08 13:55:32 ..... inserting junctions into the genome indices\n",
      "Jun 08 13:56:30 ..... started mapping\n",
      "Jun 08 13:56:31 ..... finished mapping\n",
      "Jun 08 13:56:32 ..... finished successfully\n",
      "-sample contains 3040 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -31.185568\n",
      "EM: Round [1] log likelihood: -31.185568\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n",
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 3040 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "p1=WildType_Fusion_Quantify/LowWT_HighFusion1.fq\n",
    "p2=WildType_Fusion_Quantify/LowWT_HighFusion2.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/LowWT_HighFusion --CPU $thread\n",
    "\n",
    "\n",
    "# 50% downsampling\n",
    "p1=WildType_Fusion_Quantify/LowWT_HighFusion1_0.5.fq\n",
    "p2=WildType_Fusion_Quantify/LowWT_HighFusion2_0.5.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/LowWT_HighFusion_0.5 --CPU $thread\n",
    "\n",
    "\n",
    "# 40% downsampling\n",
    "p1=WildType_Fusion_Quantify/LowWT_HighFusion1_0.4.fq\n",
    "p2=WildType_Fusion_Quantify/LowWT_HighFusion2_0.4.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/LowWT_HighFusion_0.4 --CPU $thread\n",
    "\n",
    "\n",
    "# 30% downsampling\n",
    "p1=WildType_Fusion_Quantify/LowWT_HighFusion1_0.3.fq\n",
    "p2=WildType_Fusion_Quantify/LowWT_HighFusion2_0.3.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/LowWT_HighFusion_0.3 --CPU $thread\n",
    "\n",
    "\n",
    "# 20% downsampling\n",
    "p1=WildType_Fusion_Quantify/LowWT_HighFusion1_0.2.fq\n",
    "p2=WildType_Fusion_Quantify/LowWT_HighFusion2_0.2.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/LowWT_HighFusion_0.2 --CPU $thread\n",
    "\n",
    "\n",
    "# 10% downsampling\n",
    "p1=WildType_Fusion_Quantify/LowWT_HighFusion1_0.1.fq\n",
    "p2=WildType_Fusion_Quantify/LowWT_HighFusion2_0.1.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/LowWT_HighFusion_0.1 --CPU $thread\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1a965027",
   "metadata": {},
   "source": [
    "## Similar WT and Fusion"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "id": "96659ebf",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 13:59:55 ..... started STAR run\n",
      "Jun 08 13:59:55 ..... loading genome\n",
      "Jun 08 14:00:02 ..... started 1st pass mapping\n",
      "Jun 08 14:00:10 ..... finished 1st pass mapping\n",
      "Jun 08 14:00:11 ..... inserting junctions into the genome indices\n",
      "Jun 08 14:01:09 ..... started mapping\n",
      "Jun 08 14:01:19 ..... finished mapping\n",
      "Jun 08 14:01:19 ..... finished successfully\n",
      "-sample contains 30000 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n",
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -160.499663\n",
      "EM: Round [1] log likelihood: -160.499663\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 30000 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1_0.5.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2_0.5.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1_0.5.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2_0.5.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 14:01:24 ..... started STAR run\n",
      "Jun 08 14:01:24 ..... loading genome\n",
      "Jun 08 14:01:32 ..... started 1st pass mapping\n",
      "Jun 08 14:01:36 ..... finished 1st pass mapping\n",
      "Jun 08 14:01:36 ..... inserting junctions into the genome indices\n",
      "Jun 08 14:02:35 ..... started mapping\n",
      "Jun 08 14:02:40 ..... finished mapping\n",
      "Jun 08 14:02:41 ..... finished successfully\n",
      "-sample contains 15208 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -82.446695\n",
      "EM: Round [1] log likelihood: -82.446695\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n",
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 15208 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/_starF_checkpoints\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1_0.4.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2_0.4.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1_0.4.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2_0.4.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 14:02:46 ..... started STAR run\n",
      "Jun 08 14:02:46 ..... loading genome\n",
      "Jun 08 14:02:53 ..... started 1st pass mapping\n",
      "Jun 08 14:02:57 ..... finished 1st pass mapping\n",
      "Jun 08 14:02:57 ..... inserting junctions into the genome indices\n",
      "Jun 08 14:03:56 ..... started mapping\n",
      "Jun 08 14:04:00 ..... finished mapping\n",
      "Jun 08 14:04:01 ..... finished successfully\n",
      "-sample contains 12120 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n",
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -71.389305\n",
      "EM: Round [1] log likelihood: -71.389305\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 12120 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1_0.3.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2_0.3.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1_0.3.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2_0.3.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 14:04:06 ..... started STAR run\n",
      "Jun 08 14:04:06 ..... loading genome\n",
      "Jun 08 14:04:13 ..... started 1st pass mapping\n",
      "Jun 08 14:04:16 ..... finished 1st pass mapping\n",
      "Jun 08 14:04:16 ..... inserting junctions into the genome indices\n",
      "Jun 08 14:05:15 ..... started mapping\n",
      "Jun 08 14:05:18 ..... finished mapping\n",
      "Jun 08 14:05:19 ..... finished successfully\n",
      "-sample contains 9137 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -49.795429\n",
      "EM: Round [1] log likelihood: -49.795429\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n",
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 9137 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/_starF_checkpoints\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1_0.2.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2_0.2.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1_0.2.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2_0.2.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 14:05:23 ..... started STAR run\n",
      "Jun 08 14:05:23 ..... loading genome\n",
      "Jun 08 14:05:31 ..... started 1st pass mapping\n",
      "Jun 08 14:05:33 ..... finished 1st pass mapping\n",
      "Jun 08 14:05:33 ..... inserting junctions into the genome indices\n",
      "Jun 08 14:06:32 ..... started mapping\n",
      "Jun 08 14:06:34 ..... finished mapping\n",
      "Jun 08 14:06:35 ..... finished successfully\n",
      "-sample contains 6113 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n",
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -30.453006\n",
      "EM: Round [1] log likelihood: -30.453006\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 6113 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "-ctat genome lib [/home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir] validates.\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/_starF_checkpoints\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary\n",
      "* Running CMD: /home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx  --outReadsUnmapped None  --chimSegmentMin 12  --chimJunctionOverhangMin 8  --chimOutJunctionFormat 1  --alignSJDBoverhangMin 10  --alignMatesGapMax 100000  --alignIntronMax 100000  --alignSJstitchMismatchNmax 5 -1 5 5  --runThreadN 10 --outSAMstrandField intronMotif  --outSAMunmapped Within  --alignInsertionFlush Right  --alignSplicedMateMapLminOverLmate 0  --alignSplicedMateMapLmin 30  --outSAMtype BAM Unsorted  --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1_0.1.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2_0.1.fq  --outSAMattrRGline ID:GRPundef  --chimMultimapScoreRange 3  --chimScoreJunctionNonGTAG -4  --chimMultimapNmax 20  --chimNonchimScoreDropMin 10  --peOverlapNbasesMin 12 --peOverlapMMp 0.1  --genomeLoad NoSharedMemory  --twopassMode Basic  --quantMode GeneCounts\n",
      "\t/home/jp/anaconda3/bin/STAR --genomeDir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_genome.fa.star.idx --outReadsUnmapped None --chimSegmentMin 12 --chimJunctionOverhangMin 8 --chimOutJunctionFormat 1 --alignSJDBoverhangMin 10 --alignMatesGapMax 100000 --alignIntronMax 100000 --alignSJstitchMismatchNmax 5 -1 5 5 --runThreadN 10 --outSAMstrandField intronMotif --outSAMunmapped Within --alignInsertionFlush Right --alignSplicedMateMapLminOverLmate 0 --alignSplicedMateMapLmin 30 --outSAMtype BAM Unsorted --readFilesIn /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion1_0.1.fq /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusion_Quantify/SimilarWT_Fusion2_0.1.fq --outSAMattrRGline ID:GRPundef --chimMultimapScoreRange 3 --chimScoreJunctionNonGTAG -4 --chimMultimapNmax 20 --chimNonchimScoreDropMin 10 --peOverlapNbasesMin 12 --peOverlapMMp 0.1 --genomeLoad NoSharedMemory --twopassMode Basic --quantMode GeneCounts\n",
      "\tSTAR version: 2.7.9a   compiled: 2021-05-04T09:43:56-0400 vega:/home/dobin/data/STAR/STARcode/STAR.master/source\n",
      "Jun 08 14:06:39 ..... started STAR run\n",
      "Jun 08 14:06:39 ..... loading genome\n",
      "Jun 08 14:06:47 ..... started 1st pass mapping\n",
      "Jun 08 14:06:48 ..... finished 1st pass mapping\n",
      "Jun 08 14:06:48 ..... inserting junctions into the genome indices\n",
      "Jun 08 14:07:47 ..... started mapping\n",
      "Jun 08 14:07:48 ..... finished mapping\n",
      "Jun 08 14:07:49 ..... finished successfully\n",
      "-sample contains 3040 rnaseq fragments\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.map_chimeric_reads_to_genes  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  -J Chimeric.out.junction  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt\n",
      "-building interval tree based on /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir/ref_annot.gtf.mini.sortu\n",
      "-done building interval tree (0.05 min).\n",
      "-parsing fusion evidence: Chimeric.out.junction\n",
      "-mapping reads to genes\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.handle_multimapping_reads  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --filt_file /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.fail  -M 50  > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass \n",
      "-parsing /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt, organizing read mapping info.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "done.\n",
      "\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.predict  -J /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass  -O /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion \n",
      "-parsing fusion evidence: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.junction_breakpts_to_genes.txt.pass\n",
      "-outputting fusion candidates to file: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/STAR-Fusion.filter  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary  -J Chimeric.out.junction  --tmpdir /tmp  --min_junction_reads 1  --min_sum_frags 2  --require_LDAS 1  --max_promiscuity 10  --min_pct_dom_promiscuity 20  --min_novel_junction_support 3  --min_alt_pct_junction 10  --aggregate_novel_junction_dist 5  --min_spanning_frags_only 5  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion\n",
      "CMD: mkdir -p /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir\n",
      "EM: Starting log likelihood: -15.394543\n",
      "EM: Round [1] log likelihood: -15.394543\n",
      "EM: Stopping iterations at round 1 due to insufficient improvement in likelihood.\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/../FusionFilter/blast_and_promiscuity_filter.pl --out_prefix /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion --max_promiscuity 10 --min_pct_dom_promiscuity 20  --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir \n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/blast_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/promiscuity_filter.pl --fusion_preds /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter  --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir  --max_promiscuity 10  --min_pct_dom_promiscuity 20 \n",
      "CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.pre_blast_filter.post_blast_filter.post_promisc_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter\n",
      "CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter JunctionReads,SpanningFrags > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter.abridged\n",
      "CMD: ln -sf /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.filter.intermediates_dir/star-fusion.post_blast_and_promiscuity_filter /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/incorporate_FFPM_into_final_report.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered 3040 > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/append_breakpoint_junction_info.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.filtered.FFPM /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionAnnotator/FusionAnnotator --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir --annotate /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo > /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/FusionFilter/util/filter_by_annotation_rules.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-done, see /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_likely_RT_artifacts.pl --fusions /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass --genome_lib_dir /home/jp/STAR-Fusion-v1.10.0/GRCh38_gencode_v37_CTAT_lib_Mar012021.plug-n-play/ctat_genome_lib_build_dir\n",
      "-filter_likely_RT_artifacts: (pass: 2, filtered: 0)\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/filter_by_min_FFPM.pl /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass 0.1\n",
      "* Running CMD: cp /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.preliminary/star-fusion.fusion_candidates.preliminary.wSpliceInfo.wAnnot.annot_filter.pass.RTartifact.pass.minFFPM.0.1.pass star-fusion.fusion_predictions.tsv\n",
      "* Running CMD: /home/jp/STAR-Fusion-v1.10.0/util/column_exclusions.pl star-fusion.fusion_predictions.tsv JunctionReads,SpanningFrags > star-fusion.fusion_predictions.abridged.tsv\n",
      "\n",
      "\n",
      "\t* STAR-Fusion complete.  See output: /home/jp/ManualFusions/TMPRSS2-ERG/WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.fusion_predictions.tsv (or .abridged.tsv version)\n",
      "\n",
      "\n",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "p1=WildType_Fusion_Quantify/SimilarWT_Fusion1.fq\n",
    "p2=WildType_Fusion_Quantify/SimilarWT_Fusion2.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/SimilarWT_Fusion --CPU $thread\n",
    "\n",
    "\n",
    "# 50% downsampling\n",
    "p1=WildType_Fusion_Quantify/SimilarWT_Fusion1_0.5.fq\n",
    "p2=WildType_Fusion_Quantify/SimilarWT_Fusion2_0.5.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/SimilarWT_Fusion_0.5 --CPU $thread\n",
    "\n",
    "\n",
    "# 40% downsampling\n",
    "p1=WildType_Fusion_Quantify/SimilarWT_Fusion1_0.4.fq\n",
    "p2=WildType_Fusion_Quantify/SimilarWT_Fusion2_0.4.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/SimilarWT_Fusion_0.4 --CPU $thread\n",
    "\n",
    "\n",
    "# 30% downsampling\n",
    "p1=WildType_Fusion_Quantify/SimilarWT_Fusion1_0.3.fq\n",
    "p2=WildType_Fusion_Quantify/SimilarWT_Fusion2_0.3.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/SimilarWT_Fusion_0.3 --CPU $thread\n",
    "\n",
    "\n",
    "# 20% downsampling\n",
    "p1=WildType_Fusion_Quantify/SimilarWT_Fusion1_0.2.fq\n",
    "p2=WildType_Fusion_Quantify/SimilarWT_Fusion2_0.2.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/SimilarWT_Fusion_0.2 --CPU $thread\n",
    "\n",
    "\n",
    "# 10% downsampling\n",
    "p1=WildType_Fusion_Quantify/SimilarWT_Fusion1_0.1.fq\n",
    "p2=WildType_Fusion_Quantify/SimilarWT_Fusion2_0.1.fq\n",
    "\n",
    "$starFusion --left_fq $p1 \\\n",
    "--right_fq $p2 \\\n",
    "--genome_lib_dir $gen \\\n",
    "--output_dir WildType_Fusions_Results/SimilarWT_Fusion_0.1 --CPU $thread\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "10f66e8d",
   "metadata": {},
   "source": [
    "# QC fusion results"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a1b892c5",
   "metadata": {},
   "source": [
    "## High WT Low Fusion"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "id": "c523dd10",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Non downsampled\n",
      "TMPRSS2--ERG\t63\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t52\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      "Downsampled 50%\n",
      "TMPRSS2--ERG\t32\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t29\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      "Downsampled 40%\n",
      "TMPRSS2--ERG\t23\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t21\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      " Downsampled 30%\n",
      "TMPRSS2--ERG\t21\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t17\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      " Downsampled 20%\n",
      "TMPRSS2--ERG\t13\tchr21:41494356:-\tchr21:38445621:-\n",
      "TMPRSS2--ERG\t13\tchr21:41498133:-\tchr21:38423561:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      " Downsampled 10%\n",
      "TMPRSS2--ERG\t7\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t5\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "echo \"Non downsampled\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/HighWT_LowFusion/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\nDownsampled 50%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/HighWT_LowFusion_0.5/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\nDownsampled 40%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/HighWT_LowFusion_0.4/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\n Downsampled 30%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/HighWT_LowFusion_0.3/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\n Downsampled 20%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/HighWT_LowFusion_0.2/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\n Downsampled 10%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/HighWT_LowFusion_0.1/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "71e046ee",
   "metadata": {},
   "source": [
    "## Low WT High Fusion"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "id": "fd51106e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Non downsampled\n",
      "TMPRSS2--ERG\t260\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t228\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      "Downsampled 50%\n",
      "TMPRSS2--ERG\t135\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t108\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      "Downsampled 40%\n",
      "TMPRSS2--ERG\t97\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t90\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      " Downsampled 30%\n",
      "TMPRSS2--ERG\t74\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t58\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      " Downsampled 20%\n",
      "TMPRSS2--ERG\t52\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t37\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      " Downsampled 10%\n",
      "TMPRSS2--ERG\t27\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t19\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "echo \"Non downsampled\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/LowWT_HighFusion/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\nDownsampled 50%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/LowWT_HighFusion_0.5/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\nDownsampled 40%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/LowWT_HighFusion_0.4/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\n Downsampled 30%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/LowWT_HighFusion_0.3/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\n Downsampled 20%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/LowWT_HighFusion_0.2/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\n Downsampled 10%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/LowWT_HighFusion_0.1/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e1c9bbb2",
   "metadata": {},
   "source": [
    "## Similar WT and Fusion"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "id": "b4235708",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Non downsampled\n",
      "TMPRSS2--ERG\t122\tchr21:41494356:-\tchr21:38445621:-\n",
      "TMPRSS2--ERG\t110\tchr21:41498133:-\tchr21:38423561:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      "Downsampled 50%\n",
      "TMPRSS2--ERG\t61\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t58\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      "Downsampled 40%\n",
      "TMPRSS2--ERG\t52\tchr21:41494356:-\tchr21:38445621:-\n",
      "TMPRSS2--ERG\t51\tchr21:41498133:-\tchr21:38423561:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      " Downsampled 30%\n",
      "TMPRSS2--ERG\t38\tchr21:41498133:-\tchr21:38423561:-\n",
      "TMPRSS2--ERG\t34\tchr21:41494356:-\tchr21:38445621:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      " Downsampled 20%\n",
      "TMPRSS2--ERG\t23\tchr21:41494356:-\tchr21:38445621:-\n",
      "TMPRSS2--ERG\t21\tchr21:41498133:-\tchr21:38423561:-\n",
      "\u001b[?2004h\u001b[?2004l\u001b[?2004l\n",
      " Downsampled 10%\n",
      "TMPRSS2--ERG\t14\tchr21:41494356:-\tchr21:38445621:-\n",
      "TMPRSS2--ERG\t9\tchr21:41498133:-\tchr21:38423561:-\n",
      "\u001b[?2004h"
     ]
    },
    {
     "ename": "",
     "evalue": "1",
     "output_type": "error",
     "traceback": []
    }
   ],
   "source": [
    "echo \"Non downsampled\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/SimilarWT_Fusion/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\nDownsampled 50%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/SimilarWT_Fusion_0.5/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\nDownsampled 40%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/SimilarWT_Fusion_0.4/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\n Downsampled 30%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/SimilarWT_Fusion_0.3/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\n Downsampled 20%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/SimilarWT_Fusion_0.2/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n",
    "\n",
    "echo -e \"\\n Downsampled 10%\"\n",
    "egrep \"TMPRSS2|ERG\" WildType_Fusions_Results/SimilarWT_Fusion_0.1/star-fusion.fusion_predictions.abridged.tsv|cut -f 1,2,8,10\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a490a3e9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Bash [conda env:jpEnv] *",
   "language": "bash",
   "name": "conda-env-jpEnv-bash"
  },
  "language_info": {
   "codemirror_mode": "shell",
   "file_extension": ".sh",
   "mimetype": "text/x-sh",
   "name": "bash"
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
